tests/add_server_plugins.sh

```diff
@@ -1,33 +0,0 @@
-#!/usr/bin/env bash
-
-set -e
-
-# Remove legacy folders about qgis-plugin-manager
-if [ -d /srv/plugins/.cache_qgis_plugin_manager ]; then
-  rm -rf /srv/plugins/.cache_qgis_plugin_manager
-fi
-if [ -f /srv/plugins/sources.list ]; then
-  rm -f /srv/plugins/sources.list
-fi
-
-echo "QGIS Server Lizmap and WfsOutputExtension plugins"
-echo "Unstable from https://packages.3liz.org"
-# qgis-plugin-manager init
-echo "https://packages.3liz.org/pub/server-plugins-repository/unstable/plugins.[VERSION].xml" > /tmp/sources-plugin-manager.list
```

Comment:
You must use this repository for plugins, we use dev version, and not stable version of `Lizmap server`.

Reply:
Fixed using dedicated sources file in `tests/docker-conf/unstable-plugin-sources.list` and settings the `QGIS_PLUGIN_MANAGER_SOURCES_FILE` env variable for the `map-rpc` service.